lib/state/ui/reducer.ts

```diff
@@ -22,4 +26,38 @@ const visiblePanes = (state = defaultVisiblePanes, { type, show }) => {
   return state;
 };
 
-export default combineReducers({ filteredNotes, visiblePanes });
+const defaultNote: T.NoteEntity | null = null;
```

Comment:
what's the reason for creating `defaultNote` here? I know we have a pattern occasionally of creating an empty list or object (since things like array literals are always reference-unique) but `null` is always `null` no matter which `null` it is.

Reply:
Good catch, not sure what I was doing there.